src/Models/Views/MovieViewModel.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace MvcMovie.Models.Views
+{
+    public class MovieViewModel
```

Comment:
Be careful. I think there are a few attributes that should be `Required`, for example, the Title (maybe the others, also). It depends on the business model. Use Data Annotations :)